gulpfile.babel.js

```diff
@@ -726,9 +747,7 @@ gulp.task('publish:release', ['_release'], () => {
 gulp.task('templates:styles', () => {
   return gulp.src('templates/**/*.css')
     .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
-    // FIXME: This crashes. It's a bug in gulp-csso,
-    // not csso itself.
-    //.pipe($.csso())
+    .pipe($.csso())
```

Comment:
Yup, works now. Not sure what the problem was, but it seems to have gone away with time.